src/test/java/com/blade/kit/CollectionKitTest.java

```diff
@@ -0,0 +1,65 @@
+package com.blade.kit;
+
+import java.util.HashSet;
+import java.util.ArrayList;
+import java.util.HashMap;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CollectionKitTest {
+
+  @Test
+  public void isEmpty() {
+    final Object[] array = {};
+    final Object[] array2 = {null};
+    final ArrayList collection = new ArrayList();
+    final ArrayList collection2 = new ArrayList();
+    collection2.add(null);
+
+    Assert.assertTrue(CollectionKit.isEmpty(array));
+    Assert.assertFalse(CollectionKit.isEmpty(array2));
+    Assert.assertTrue(CollectionKit.isEmpty(collection));
+    Assert.assertFalse(CollectionKit.isEmpty(collection2));
```

Comment:
Add test for `isEmpty(null)`

Reply:
`isEmpty(null)` results in a compile error because of an ambiguous method call but this works 

`final Object[] array3 = null;
  Assert.assertTrue(CollectionKit.isEmpty(array3));`